packages/babel-preset-env/data/built-in-modules.json

```diff
@@ -6,7 +6,7 @@
     "safari": "10.1",
     "opera": "48",
     "ios_saf": "10.3",
-    "and_chr": "71",
-    "and_ff": "64"
+    "and_chr": "74",
```

Comment:
Is this technically a breaking change? A script which might have worked on and_chr 72 now it won't work anymore.

Reply:
Reverted this change and will push a new PR that updates the script to only update if it results in an earlier (or new) target